templates/ts-simple/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@scramjet/ts-simple",
+  "version": "1.0.0",
+  "description": "Scramjet Cloud Platform Sequence Typescript Template",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
scripts?

```suggestion
  "scripts": {
    "build": "npm i && tsc -p tsconfig.json",
    "postbuild": "cp -r package.json dist/ && (cd dist && npm i --only=production)",
    "pack": "si seq pack ./dist/",
    "clean": "rm -rf ./dist ./*.tar.gz"
  },
```

Reply:
Inconsistency between languages with different scirpts- for now leaving it empty as a request issue:
https://github.com/scramjetorg/create-sequence/issues/3